### ISSUE TITLE
Customer needs an address on subsequent recurring payment

### ISSUE THREAD

**Issue by CasEbb** [MultiSafepay/php-sdk#7]
The following code:
```php
$customerDetails = (new CustomerDetails())
    ->addReference($customerId);

$paymentOptions = (new PaymentOptions())
    ->addRedirectUrl("https://www.example.com/webhooks/payment");

$orderRequest = (new OrderRequest())
    ->addType('direct')
    ->addOrderId($orderId)
    ->addDescriptionText($description)
    ->addMoney($amount)
    ->addCustomer($customerDetails)
    ->addPaymentOptions($paymentOptions)
    ->addRecurringModel(OrderRequest::UNSCHEDULED_MODEL)
    ->addRecurringId($recurringId);

$multiSafepaySdk->getTransactionManager()->create($orderRequest);
```

Results in an exception:
```
An uncaught Exception was encountered

Type:        Error
Message:     Call to a member function getStreetName() on null
Filename:    /var/www/html/tradingtoppers.nl/application/vendor/multisafepay/php-sdk/src/Api/Transactions/OrderRequest/Arguments/CustomerDetails.php
Line Number: 60
```

CustomerDetails::getData tries to serialize the address, but it's absent. As this is a subsequent recurring payment, the Customer's address details should already be known (using the customer reference).

**Comment by Daniel-MultiSafepay** [comment]
Hi @CasEbb. 

Thanks for contacting us and reporting this issue. 
We will review what the problem is and get back to you with more information.

**Comment by Daniel-MultiSafepay** [comment]
Hi @CasEbb 

We just released a new version (5.11.2) where this issue should be now fixed. You should be able to process an order request like the one in your example. 

Please, feel free to reopen this issue, or create a new one if this is still not working for you.
